src/CMakeLists.txt

```diff
@@ -66,14 +75,24 @@ endif(ALLOW_UNSAFE_UPROBE)
 
 target_link_libraries(bpftrace arch ast parser resources)
 
-if (STATIC_LINKING)
+if(STATIC_LINKING)
   target_link_libraries(bpftrace ${LIBBCC_LIBRARIES})
   target_link_libraries(bpftrace ${LIBBPF_LIBRARY_STATIC})
   target_link_libraries(bpftrace ${LIBBCC_LOADER_LIBRARY_STATIC})
+
+  add_library(LIBELF STATIC IMPORTED)
+  set_property(TARGET LIBELF PROPERTY IMPORTED_LOCATION ${LIBELF_LIBRARIES})
+  target_link_libraries(bpftrace LIBELF)
+
+  if(NOT STATIC_LIBC)
+    if(EMBED_LLVM OR EMBED_CLANG)
```

Comment:
Could put this on one line:
```
if(NOT STATIC_LIBC AND (EMBED_LLVM OR EMBED_CLANG))
```

Reply:
Thanks, I wasn't familiar with compound expressions in cmake so I played it safe while I was prototyping. I'll adjust it.

Reply:
It turns out that the latter conditions aren't needed anyways.